web/public/index.html

```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
 
-    <link rel="icon" type="image/png" href="maiden_16.png" sizes="16x16">  
-    <link rel="icon" type="image/png" href="maiden_32.png" sizes="32x32">  
-    <link rel="icon" type="image/png" href="maiden_96.png" sizes="96x96">  
+    <link rel="icon" type="image/png" href="/maiden/maiden_16.png" sizes="16x16">  
+    <link rel="icon" type="image/png" href="/maiden/maiden_32.png" sizes="32x32">  
+    <link rel="icon" type="image/png" href="/maiden/maiden_96.png" sizes="96x96">  
```

Comment:
If we used fragments in the URLs I think we could bypass the this change and retain the ability to host maiden from an arbitrary root

Reply:
Yeah, I think the fragments idea solves a lot of that and would eliminate the need for a lot of these auxiliary changes.

Reply:
Removed all changes here.